File: sample-chat-obj-c/sample-chat/InfoScreen/InfoTableViewCell/InfoTableViewCell.m
//
//  InfoTableViewCell.m
//  sample-conference-videochat
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "InfoTableViewCell.h"

@implementation InfoTableViewCell

- (void)applyInfo:(InfoModel*)model {
    self.titleInfoLabel.text = model.title;
    self.descriptInfoLabel.text = model.info;
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Components/InfoScreenViewController/InfoTableViewCell.swift
//
//  InfoTableViewCell.swift
//  sample-conference-videochat-swift
//
//  Created by Vladimir Nybozhinsky on 12/30/18.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class InfoTableViewCell: UITableViewCell {

    @IBOutlet weak var titleInfoLabel: UILabel!
    @IBOutlet weak var descriptInfoLabel: UILabel!
    
    func applyInfo(model: InfoModel) {
        titleInfoLabel.text = model.title
        descriptInfoLabel.text = model.info
    }
}
